ev3dev2/port.py

```diff
@@ -98,15 +98,15 @@ def driver_name(self):
         Returns the name of the driver that loaded this device. You can find the
         complete list of drivers in the [list of port drivers].
         """
-        self._driver_name, value = self.get_attr_string(self._driver_name, 'driver_name')
+        (self._driver_name, value) = self.get_cached_attr_string(self._driver_name, 'driver_name')
```

Comment:
Are you sure this can be cached? Aren't you able to change the driver _using this class_?

Reply:
This is the driver name for the port driver. It does not change. You are probably thinking of `set_device` which sets the sensor that is connected to the port.